### DISCUSSION TITLE
Pages for ELKS PC-98 in Japanese

### DISCUSSION THREAD

**Discussion by tyama501** [jbruchon/elks#1435] [Show and tell]
Hello,

To increase users in Japan I made a page introducing ELKS PC-98 in Japanese.
https://tyama501.github.io/elks98doc/

**Comment by toncho11** [comment]
So you have a working X-nano driver for PC-98?

**Reply by tyama501** [reply]
I still planing to add clearing screen when opening but yes it is working for the graphic.
https://twitter.com/i/status/1564683753254924288

For mouse, I need to write new driver since it is not serial mouse.

**Comment by tyama501** [comment]
I have updated my web site with the nice desktop image on ELKS 0.9.0.

https://tyama501.github.io/elks98doc/

**Reply by toncho11** [reply]
Thank you @tyama501 for the screenshot!!! :)
You can also add: https://github.com/toncho11/microwindows/wiki It explains things like how an exisiting /tmp/nxsock can block nxdsktop starting. Actually this might be considered a bug in Microwindows - not able to start should not leave this file in /tmp. The link also acts as a documentation for nxdsktop.

**Reply by tyama501** [reply]
Hello @toncho11 

Thank you! I have added the link and the note.

**Comment by floriangit** [comment]
ページはよさそう! ありがとうございました.

**Reply by tyama501** [reply]
ありがとうございます！
